packages/edit-post/src/components/layout/index.js

```diff
@@ -163,6 +164,9 @@ function Layout() {
 		},
 		[ entitiesSavedStatesCallback ]
 	);
+	const ref = useRef();
+
+	useEditorStyles( ref, settings.styles );
```

Comment:
Looks like this broke the "use theme editor styles" option, it's now always using the theme styles.

Reply:
(I have a fix coming)